tools/symbols_tool/symbols_tool.py

```diff
@@ -0,0 +1,100 @@
+# Lint as: python2, python3
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A wrapper for the `symbols` tool, that can handle multiple inputs."""
+
+import argparse
+import os
+import re
+import sys
+
+from build_bazel_rules_apple.tools.wrapper_common import execute
+
+_TIMING_INFO_PATTERN = r"(.*\s\[.*,\s.*\sseconds\]:)"
+
+
+def _generate_symbols(args):
+  """Generate argument parser for tool."""
+  output_dir = args.output_dir
+  archs = args.arch
+  binaries = args.binary
+
+  if not os.path.exists(output_dir):
+    os.makedirs(output_dir)
+
+  cmd = ["/bin/bash", "-c"]
+  symbols_cmds = []
+  for i, arch in enumerate(archs):
+    symbols_cmds += ["xcrun symbols -noTextInSOD -noDaemon -arch {arch} \
+        -symbolsPackageDir {output_dir} {binary}".format(
+            arch=arch,
+            output_dir=output_dir,
+            binary=binaries[i],
+        )]
+  cmd.append("\n".join(symbols_cmds))
+  execute.execute_and_filter_output(
```

Comment:
instead of executing the list of commands via bash, I think it'd be preferable to execute each of them separately? If not, we'll need to add a `set -e` and potentially handle quoting in the `xcrun` invocation to ensure that (a) failure of any command will exit non-0 and (b) spaces wont break things